lib/unison-sqlite/src/Unison/Sqlite/Connection.hs

```diff
@@ -58,16 +58,26 @@ module Unison.Sqlite.Connection
   )
 where
 
+import qualified Data.Text as Text
+import qualified Data.Text.IO as Text
 import qualified Database.SQLite.Simple as Sqlite
 import qualified Database.SQLite.Simple.FromField as Sqlite
 import qualified Database.SQLite3.Direct as Sqlite (Database (..))
 import Debug.RecoverRTTI (anythingToString)
+import System.IO (stderr)
 import Unison.Prelude
 import Unison.Sqlite.Exception
 import Unison.Sqlite.Sql
 import UnliftIO (MonadUnliftIO)
+import System.IO.Unsafe (unsafePerformIO)
+import System.Environment (lookupEnv)
 import UnliftIO.Exception
 
+debugTraceQueries :: Bool
+debugTraceQueries =
+  isJust (unsafePerformIO (lookupEnv "UNISON_SQLITE_DEBUG"))
```

Comment:
```suggestion
  maybe False (not . null) (unsafePerformIO (lookupEnv "UNISON_SQLITE_DEBUG"))
```

I suppose it's a matter of taste, but I'd prefer that we treat `UNISON_SQLITE_DEBUG=""` as falsy.

Reply:
Hmm... I'm on the fence. I think you could just not pass the env var at all if you don't want to debug ;)

If `""` is falsy, then we can't do `UNISON_SQLITE_DEBUG= stack run`, which is the least-friction way. Anything else requires making up a random string. I'm leaning towards leaving it.

Reply:
I usually pass `UNISON_SQLITE_DEBUG=true` for this sort of thing, but that has its own problems of implying you can pass `UNISON_SQLITE_DEBUG=false`, which won't work.

I'm not terribly opinionated on this one, so feel free to leave it as is 👍🏼